pmd-ui/src/main/resources/net/sourceforge/pmd/util/fxdesigner/fxml/xpath.fxml

```diff
@@ -28,24 +58,72 @@
                 </AnchorPane>
                 <AnchorPane>
                     <children>
-                        <TitledPane collapsible="false" styleClass="accent-header" text="XPath Expression" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
+                        <ToolbarTitledPane fx:id="expressionTitledPane"
+                                           styleClass="accent-header"
+                                           collapsible="false"
+                                           title="XPath Expression"
+                                           AnchorPane.bottomAnchor="0.0"
+                                           AnchorPane.leftAnchor="0.0"
+                                           AnchorPane.rightAnchor="0.0"
+                                           AnchorPane.topAnchor="0.0">
+                            <toolbarItems>
+                                <MenuButton mnemonicParsing="false" styleClass="menu-button-no-arrow,icon-button">
+                                    <graphic>
+                                        <FontIcon iconLiteral="fa-cog" />
+                                    </graphic>
+                                    <tooltip>
+                                        <Tooltip>
+                                            <text>
+                                                <String fx:value="Configure XPath version" />
+                                            </text>
+                                        </Tooltip>
+                                    </tooltip>
+                                    <items>
+                                        <fx:define>
+                                            <ToggleGroup fx:id="xpathVersionToggleGroup" />
+                                        </fx:define>
+                                        <RadioMenuItem mnemonicParsing="false"
+                                                       text="XPath 1.0"
+                                                       userData="1.0"
```

Comment:
the old approach had the values taken from the properties themself... hardwiring it here seems like a small step back in maintainability

Reply:
I see what you mean, that's corrected